src/include/fips.h

```diff
@@ -50,10 +50,17 @@ static const int FIPS_BLACKLIST[] = {DES_ECB, DES_CBC, DES_CBC_CS, DES_OFB,
     DES3_CBC, DES3_CBC_CS, DES3_OFB, DES3_CFB, DES3_CTR, DES3_CTRLST,
     DES3_CBC_MAC, DES3_CMAC, ED25519_KEYGEN, ED25519_SIGN, ED25519_VERIFY,
     ED448_KEYGEN, ED448_SIGN, ED448_VERIFY, X25519_KEYGEN, X25519_DERIVE,
-    X448_KEYGEN, X448_DERIVE };
+    X448_KEYGEN, X448_DERIVE, RSA_ME, RSA_CRT, SHA512_DRNG };
 static const size_t FIPS_BLACKLIST_LEN = sizeof(FIPS_BLACKLIST)
     / sizeof(FIPS_BLACKLIST[0]);
 
+/*
+ * FIPS service indicator: List of tolerated but non-approved algorithms.
+ */
+static const int FIPS_OVERRIDE_LIST[] = { RSA_ME, RSA_CRT, SHA512_DRNG };
+static const size_t FIPS_OVERRIDE_LIST_LEN = sizeof(FIPS_OVERRIDE_LIST)
+	/ sizeof(FIPS_OVERRIDE_LIST[0]);
```

Comment:
I would place the / on the first line, but thats just my personal opinion.....

Reply:
I agree ... I never liked the coding in lines 76 + 77, which is old code, but wanted to add the new code in the same way. OK, this is the trigger to change both places ;-)

Reply:
Update: I would code it like:
```
static const size_t FIPS_OVERRIDE_LIST_LEN
	= sizeof(FIPS_OVERRIDE_LIST) / sizeof(FIPS_OVERRIDE_LIST[0]);
```
Looks good to me, but this is also personal preference ...

Reply:
I would place the = at the end of the first line, but other than that it looks good.